### PR TITLE
Whats new feature modal

### PR DESCRIPTION
**Pull request by ShaileshParmar-WebDeveloper**
### Describe your changes :

I worked on the Whats new modal.

#
### Type of change :

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
- What's new CTA in nav bar with icon
image
image
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @shahsank3t, @darth-coder00, @Sachin-chaurasiya

### CODE REVIEW

File: catalog-rest-service/src/main/resources/ui/package.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "immutable": "^4.0.0-rc.14",
     "jquery": "^3.5.0",
     "markdown-draft-js": "^2.3.0",
+    "markdown-loader": "^6.0.0",
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/package.json:47]: Why we need this?

File: catalog-rest-service/src/main/resources/ui/src/components/app-bar/Appbar.tsx [hunk 2]
```diff
@@ -178,6 +191,12 @@ const Appbar: React.FC = (): JSX.Element => {
               />
             </div>
           </div>
+          {isModalOpen && (
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/src/components/app-bar/Appbar.tsx:194]: Use self-descriptive variable names. For instance, `isWhatsNewModalOpen` or `isFeatureModalOpen`

File: catalog-rest-service/src/main/resources/ui/tailwind.config.js [hunk 3]
```diff
@@ -37,6 +37,7 @@ const bodyBG = '#FCFBFE';
 const bodyHoverBG = '#F9F8FD';
 const tagBG = '#EEEAF8';
 const primaryBG = '#7147E840'; // 'rgba(113, 71, 232, 0.25)';
+const greyBG = '#302E36';
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/tailwind.config.js:40]: name it backdropBG

File: catalog-rest-service/src/main/resources/ui/tailwind.config.js [hunk 4]
```diff
@@ -77,6 +77,7 @@ module.exports = {
         'grey-muted': textMuted,
         'grey-muted-lite': textMutedLite,
         'grey-dark': textDark,
+        'grey-lite': greyBG,
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/tailwind.config.js:80]: name it grey-backdrop

File: catalog-rest-service/src/main/resources/ui/webpack.config.dev.js [hunk 5]
```diff
@@ -47,7 +47,7 @@ module.exports = {
       // .js and .jsx files to be handled by babel-loader
       {
         test: /\.(js|jsx)$/,
-        exclude: /node_modules/,
+        // exclude: /node_modules/,
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/webpack.config.dev.js:50]: Why we need to include this?

File: catalog-rest-service/src/main/resources/ui/webpack.config.dev.js [hunk 6]
```diff
@@ -89,7 +89,7 @@ module.exports = {
             },
           },
         ],
-        exclude: /node_modules/, // Just the source code
+        // exclude: /node_modules/, // Just the source code
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/webpack.config.dev.js:92]: Why this?

File: catalog-rest-service/src/main/resources/ui/webpack.config.dev.js [hunk 7]
```diff
@@ -103,7 +103,7 @@ module.exports = {
...
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/webpack.config.dev.js:106]: Same here

**Review by darth-coder00 — CHANGES_REQUESTED** [review]
Just a couple of changes and queries, Otherwise it's looking good!

File: catalog-rest-service/src/main/resources/ui/webpack.config.dev.js (hunk 5 above)
Comment (ShaileshParmar-WebDeveloper) [reply catalog-rest-service/src/main/resources/ui/webpack.config.dev.js:50]: for react-slick webpack is throwing error while parsing the files, after including node_module its running properly

File: catalog-rest-service/src/main/resources/ui/webpack.config.dev.js (hunk 6 above)
Comment (ShaileshParmar-WebDeveloper) [reply catalog-rest-service/src/main/resources/ui/webpack.config.dev.js:92]: same for here

File: catalog-rest-service/src/main/resources/ui/webpack.config.dev.js (hunk 7 above)
Comment (ShaileshParmar-WebDeveloper) [reply catalog-rest-service/src/main/resources/ui/webpack.config.dev.js:106]: and here as well

File: catalog-rest-service/src/main/resources/ui/webpack.config.dev.js (hunk 5 above)
Comment (darth-coder00) [reply catalog-rest-service/src/main/resources/ui/webpack.config.dev.js:50]: We don't need to include node_modules for js | jsx files

Comment (ShaileshParmar-WebDeveloper) [reply catalog-rest-service/src/main/resources/ui/webpack.config.dev.js]: updated

**Pull request merged**